forem/forem
# Focus trap for Modals

## What type of PR is this? (check all applicable)


## Description

This PR adds FocusTrap to our modals making them more accessible.

## Related Tickets & Documents

- Closes #12427

## QA Instructions, Screenshots, Recordings

1. Go to /new
2. Open modal by clicking link in the sidebar
3. When you next hit `[TAB]` it should focus on something inside modal (previously it was still going through links in the sidebar behind modal).

## Added tests?

- [x] No, not needed.

## Added to documentation?

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/)
